client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/RPCProblemDto.java

```diff
@@ -32,6 +33,6 @@ public class RPCProblemDto extends ProblemInfoDto{
     /**
      * a list of seeded RPC tests
      */
-    public List<List<RPCActionDto>> seededTestDtos;
+    public Map<String, List<RPCActionDto>> seededTestDtos;
```

Comment:
what is the key here? add comment

Reply:
test case name which is used to provide warning info